package.json

```diff
@@ -37,7 +37,6 @@
     "jsdoc-to-markdown": "^2.0.1",
     "mocha": "^3.2.0",
     "mochainon": "^1.0.0",
-    "wary": "^1.1.0"
```

Comment:
> AFAIK the output-changing shouldn't matter

Well, if the output != expected output, it does. And in this case it would go undetected, as the tests didn't explicitly check, and there was no way to inspect the output post-mortem.

> so that all the "big writes" happen in a temporary directory, rather than on the local filesystem?

I'm not sure I follow; Temporary files / directories are on the local file system, just not in the repository.